elephant/causality/granger.py

```diff
@@ -14,18 +14,30 @@
 prediction error of another signal, compared to the past values of the latter
 signal alone. If it does reduce the prediction error, the first signal is said
 to Granger cause the other signal.
+Granger causality analysis can be extended to the spectral domain investigating
+the influnece signals have onto each other in a frequency resolved manner. It
+relies on estimating the cross-spectrum of time series and decomposing them
+into a transfer function and a noise covariance matrix. The method to estimate
+the spectral Granger causality is non-parametric in the sense that it does not
+require to fit an autoregressive model (see :cite:`granger-Dhamala02_354`).
 
 Limitations
 -----------
 The user must be mindful of the method's limitations, which are assumptions of
 covariance stationary data, linearity imposed by the underlying autoregressive
 modelling as well as the fact that the variables not included in the model will
 not be accounted for :cite:`granger-Seth07_1667`.
+When estimating spectral Granger causality the user must be familiar with
+basics the multitaper method to estimate power- and cross-spectra (e.g.
+sampling frequency, DPSS, time-half bandwidth product).
 
 Implementation
 --------------
 The mathematical implementation of Granger causality methods in this module
 closely follows :cite:`granger-Ding06_0608035`.
+The implementation of spectral Granger causality follows
+:cite:`granger-Dhamala02_354`, :cite:`granger-Wend13_20110610` and
+:cite:`granger-Wilson72` for the spectral matrix factorization.
```

Comment:
These references do not seem to have been included to the bibtex [elephant.bib](https://github.com/NeuralEnsemble/elephant/blob/03275d39c00acd60eaa21472ec2b0851f59cbbbe/doc/bib/elephant.bib)

Reply:
Addressed in [d1f0dc8](https://github.com/NeuralEnsemble/elephant/pull/545/commits/d1f0dc8094012587c9784ffc275c270ed492a3bc).

Reply:
Great!